src/screen/HtmlScreen.js

```diff
@@ -329,19 +373,25 @@ class HtmlScreen extends RequestScreen {
 
 }
 
+/**
+ * Helper selector for ignore favicon when exist data-senna-track.
+ */
+const ignoreFavicon = ':not([rel="Shortcut Icon"]):not([rel="shortcut icon"]):not([rel="icon"]):not([href$="favicon.icon"])';
```

Comment:
I decided to just ignore the favicon so I do not have to add `link` use cases. We support some in addition to `stylesheet` within the portal, so I'm not sure what we're really enduring.  I don't want to risk breaking-change.